server/planning/coverage.py

```diff
@@ -243,7 +311,18 @@ def _set_assignment_information(self, doc):
             'name': {'type': 'string'},
             'label': {'type': 'string'}
         }
-    }
+    },
+    'assigned_to': {
+        'type': 'dict',
+        'mapping': {
+            'type': 'object',
+            'properties': {
+                'assignment_id': not_analyzed,
```

Comment:
is this supposed to be some sort of current assignment?

Reply:
It points to the assignment and on `GET` assignment information is added to the coverage.